.github/workflows/release.yml

```diff
@@ -1,6 +1,12 @@
 name: Release
 
-on: ['push']
+on:
```

Comment:
This mean that we won't have docker builds for non master branch right ?

Reply:
That's what the new code means, yes. Of course, we can set it up however we want.